include/bitcoin/system/math/ec_scalar.hpp

```diff
@@ -32,17 +33,21 @@ class BC_API ec_scalar
 
     /// Constructors.
     ec_scalar();
+    explicit ec_scalar(uint64_t value);
     ec_scalar(const ec_secret& secret);
+    ec_scalar(const ec_scalar& scalar);
+    ec_scalar(ec_scalar&& scalar);
+    ~ec_scalar() = default;
```

Comment:
This is explicitly provided because of the Rule of 5. We have copy constructor move constructor, copy operator and move operator. Ro5 states there should also be a destructor. So in this case, I instantiate the default one.

Reply:
Why would it be necessary to make it explicit to comply with this guideline? Not having an explicit destructor implies a default destructor. We don't do this in any class, so I would opt for consistency over adherence to a distinction without a difference.